pyproject.toml

```diff
@@ -1,3 +1,268 @@
+[project]
```

Comment:
shame this ends up in pyproject.toml -- now this file is super bloated and difficult to use :S

Reply:
Yeah... we have no other choice though